src/Apps/Order/redirects.tsx

```diff
@@ -1,13 +1,20 @@
 import { routes_OrderQueryResponse } from "__generated__/routes_OrderQuery.graphql"
 import { Location, RouteConfig, Router } from "found"
 import moment from "moment"
+import { graphql } from "react-relay"
 import { get } from "Utils/get"
 import { RedirectPredicate, RedirectRecord } from "./getRedirect"
 import { OrderApp } from "./OrderApp"
 
+import { redirects_order } from "__generated__/redirects_order.graphql"
+
 const LEAVE_MESSAGING =
   "Are you sure you want to refresh? Your changes will not be saved."
 
+interface OrderQuery {
+  order: redirects_order
+}
```

Comment:
Instead of defining this interface and having to repeat `RedirectPredicate<OrderQuery>` throughout the file, you could also define something like:

```ts
type OrderPredicate = RedirectPredicate<{ order: redirects_order }>

const goToArtworkIfOrderWasAbandoned: OrderPredicate = ({ order }) => {
  // ...
}
```